libs/hdf_converters/test/test.ts

```diff
@@ -0,0 +1,149 @@
+import fs from 'fs';
+import {ExecJSON} from 'inspecjs';
+import {BurpSuiteMapper} from '../src/burpsuite-mapper';
+import {JfrogXrayMapper} from '../src/jfrog-xray-mapper';
+import {NiktoMapper} from '../src/nikto-mapper';
+import {SarifMapper} from '../src/sarif-mapper';
+import {ScoutsuiteMapper} from '../src/scoutsuite-mapper';
+import {XCCDFResultsMapper} from '../src/xccdf-results-mapper';
+import {ZapMapper} from '../src/zap-mapper';
+
+test('Returns proper Hdf output from Burpsuite', () => {
+  const mapper = new BurpSuiteMapper(
+    fs.readFileSync(
+      '/Users/rlin/Desktop/Repositories/heimdall_tools/sample_jsons/burpsuite_mapper/sample_input_report/zero.webappsecurity.com.min',
```

Comment:
You need to use relative paths here, absolute paths will only work on your system.